Sources/Brave/Frontend/Browser/Helpers/ErrorPageHelper.swift

```diff
@@ -135,18 +135,26 @@ class ErrorPageHelper {
 }
 
 extension ErrorPageHelper {
-  static func certificateError(for url: URL) -> Int {
+  static func errorCode(for url: URL) -> Int {
     if InternalURL.isValid(url: url),
       let internalUrl = InternalURL(url),
-      internalUrl.isErrorPage {
-
+       internalUrl.isErrorPage {
+      
       let query = url.getQuery()
       guard let code = query["code"],
-        let errCode = Int(code)
+            let errCode = Int(code)
       else {
         return 0
       }
-
+      
+      return errCode
+    }
+    return 0
+  }
+  
+  static func certificateError(for url: URL) -> Int {
+    let errCode = errorCode(for: url)
+    if errCode != 0 {
```

Comment:
Can you add comment here what does error code 0 mean and why do we limit that code?

Reply:
It means there's none. When there is an errorCode, `CFNetworkError(rawValue: ...)` minimum value is `1`, not `0`. So `0 = errSecNo`. IE: 0 = no error.